packages/twenty-front/src/pages/auth/SignInUp.tsx

```diff
@@ -39,10 +41,18 @@ export const SignInUp = () => {
   return (
     <>
       <AnimatedEaseIn>
-        <Logo />
+        {signInUpStep === SignInUpStep.SSOWorkspaceSelection ? (
+          <IconLockCustom size={40} />
+        ) : (
+          <Logo />
+        )}
       </AnimatedEaseIn>
       <Title animate>{title}</Title>
-      <SignInUpForm />
+      {signInUpStep === SignInUpStep.SSOWorkspaceSelection ? (
+        <SSOWorkspaceSelection />
```

Comment:
looks wrong, SignInUpForm is already making a lot of switch case based on SignInUpStep

Reply:
What's wrong here?